pkgs/games/r2mod_cli/default.nix

```diff
@@ -0,0 +1,48 @@
+{ fetchFromGitHub
+, jq
+, makeWrapper
+, p7zip
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "r2mod_cli";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "Foldex";
+    repo = "r2mod_cli";
+    rev = "v${version}";
+    sha256 = "1g64f8ms7yz4rzm6xb93agc08kh9sbwkhvq35dpfhvi6v59j3n5m";
+
+  };
+
+  buildInputs = [
+    jq
+    makeWrapper
+    p7zip
+  ];
+
+  phases = [ "installPhase" ];
+
+  installPhase = ''
+    mkdir -p "$out/bin"
+    mkdir -p "$out/share/bash-completion/completions"
+    mkdir -p "$out/share/zsh/site-functions"
+    install -m 755 "$src/r2mod" "$out/bin/r2mod"
+    install -m 644 "$src/completions/bash/r2mod.sh" "$out/share/bash-completion/completions/r2mod"
+    install -m 644 "$src/completions/zsh/_r2mod" "$out/share/zsh/site-functions/_r2mod"
```

Comment:
You can combine all of those with `install -D`, which creates the missing directories

Reply:
Thanks for pointing this out. :+1: